builder/builder_test.go

```diff
@@ -114,13 +114,14 @@ func TestBuild(t *testing.T) {
 			)
 
 			payload := &builder.Payload{
-				Transactions: bfttypes.ToTxs(inclusionListTxs),
-				GasLimit:     0,
-				Timestamp:    g.Time + 1,
-				NoTxPool:     test.noTxPool,
+				InjectedTransactions: bfttypes.ToTxs(inclusionListTxs),
+				GasLimit:             0,
+				Timestamp:            g.Time + 1,
+				NoTxPool:             test.noTxPool,
 			}
 			preBuildInfo := app.Info(abcitypes.RequestInfo{})
-			require.NoError(t, b.Build(payload))
+			_, err = b.Build(payload)
```

Comment:
We should add a test ensuring that the block returned here matches `wantBlock` below.

`require.Equal(t, wantBlock, builtBlock)` or something.

Reply:
Good catch. 216237b